screens/Authentication.jsx

```diff
@@ -0,0 +1,95 @@
+import React, {useState} from 'react';
+import {
+  Button,
+  StyleSheet,
+  Text,
+  TextInput,
+  TouchableOpacity,
+  View,
+} from 'react-native';
+
+const Authentication = () => {
+  const [phone_number, setPhoneNumber] = useState('');
+  const handlePhoneNumberChange = number => {
+    const formatted_number = number.replace(/[^0-9]/g, '');
+    if (number.length <= 10) {
+      setPhoneNumber(formatted_number);
+    }
+  };
+  return (
+    <View style={styles.container}>
+      <View>
+        <Text style={styles.title}>Verify your mobile number </Text>
+        <Text style={styles.subtitle}>
+          Enter your number so that we know you're you
```

Comment:
move these copies to some constant folder

Reply:
can you elaborate more on this